.github/workflows/contract-integration.yaml

```diff
@@ -20,16 +20,42 @@ jobs:
           python-version: 3
       - run: |
           pip install -e .
-          cd provider/*project
+          cd lookoutvision/*project
           curl -s -L https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip -o sam.zip
           unzip -q sam.zip -d sam-install
           sudo ./sam-install/install
           sam local start-lambda &
-          pip install --quiet cloudformation-cli-java-plugin
+          pip install --quiet cloudformation-cli-java-plugin cloudformation-cli-python-plugin
```

Comment:
Does this need both the java plugin and the python plugin?

Reply:
Neither of these jobs need both plugins. Just kept the same line in both jobs in the hopes of being able to re-use some code next time we continue expanding the languages tested